tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -754,6 +754,8 @@ private void SetupBuildkiteEnvironment()
             Message = EnvironmentHelpers.GetEnvironmentVariable("BUILDKITE_MESSAGE");
             AuthorName = EnvironmentHelpers.GetEnvironmentVariable("BUILDKITE_BUILD_AUTHOR");
             AuthorEmail = EnvironmentHelpers.GetEnvironmentVariable("BUILDKITE_BUILD_AUTHOR_EMAIL");
+            CommiterEmail = EnvironmentHelpers.GetEnvironmentVariable("BUILDKITE_BUILD_CREATOR");
```

Comment:
```suggestion
            CommitterName = EnvironmentHelpers.GetEnvironmentVariable("BUILDKITE_BUILD_CREATOR");
```

Reply:
Oops! I though I had fixed the typo everywhere but I didn't.